README.md

```diff
@@ -20,7 +19,7 @@ uphold this code.
 We use [GitHub issues](https://github.com/Qiskit/qiskit-experiments/issues) for
 tracking requests and bugs. Please
 [join the Qiskit Slack community](https://ibm.co/joinqiskitslack)
-and use our [Qiskit Slack channel](https://qiskit.slack.com) for discussion and
+and use the `#experiments` channel in the [Qiskit Slack workspace](https://qiskit.slack.com) for discussion and
```

Comment:
You could also link directly to the channel (https://qiskit.slack.com/archives/CGZDF48EN) but maybe that's overkill.

Reply:
Oh nice, that's better, thanks.